astropy/_erfa/setup_package.py

```diff
@@ -31,9 +32,25 @@ def pre_build_ext_hook(cmd_obj):
 def pre_sdist_hook(cmd_obj):
     preprocess_source()
 
+def is_release():
+    """
+    Determines if we're on a release version of astropy
+    """
```

Comment:
Could also use `astropy_helpers.version_helpers.get_pkg_version_module` but that seems like overkill too.